sys/mips/conf/std.CHERI_PURECAP

```diff
@@ -0,0 +1,50 @@
+#
+# CHERI -- Kernel for the SRI/Cambridge "CHERI" (Capability Hardware Enhanced
+# RISC Instructions) FPGA soft core, as configured in its Terasic DE-4
+# reference configuration - 128bit version.
+#
+# $FreeBSD$
+#
+
+include		"std.CHERI"
+
+machine		mips mips64c128
+
+nomakeoptions 	ARCH_FLAGS
+makeoptions 	ARCH_PIC_FLAGS="-fpic"
+makeoptions 	ARCH_FLAGS="-mcpu=cheri128 -mxcaptable"
+makeoptions 	CFLAGS+="-gdwarf-5"
+makeoptions 	WERROR="-Wno-error"
+makeoptions 	KERNLOADADDR=0xffffffff80100000
+makeoptions 	CHERI_USE_CAP_TABLE=plt
+makeoptions	CHERI_SUBOBJECT_BOUNDS=subobject-safe
+makeoptions   LDSCRIPT_NAME=ldscript.mips.cheri
+
+# purecap kernel options
+options		CHERI_PURECAP_KERNEL
+options 	KSTACK_LARGE_PAGE
+options   NO_SWAPPING
+
+# debugging
+options 	KTRACE
+options 	KDB
+options		KDB_TRACE # stacktrace on panic
+nooptions	DDB
+
+# disable raid, we don't need it
+nooptions	GEOM_RAID
+
+# networking
+options		INET6
+device		tuntap
+
+# filesystems
+options 	TMPFS
+options		FFS
+options		CD9660
+options		MSDOSFS
+nooptions		NFSD, NFSCL
+
+options		HWPMC_HOOKS
+device		hwpmc
+device		hwpmc_beri
```

Comment:
All of this (except the nooptions) belongs in a base MIPS/BERI/CHERI config, it's not purecap-specific.